src/prefabs/link-hover-menu.tsx

```diff
@@ -0,0 +1,42 @@
+/** @jsx createElementEntity */
+import { ArrayVec3, Attrs, createElementEntity, createRef } from "../utils/jsx-entity";
+import { BUTTON_TYPES, Button3D } from "./button3D";
+
+const BUTTON_HEIGHT = 0.2;
+const BUTTON_WIDTH = 0.6;
+const BUTTON_SCALE: ArrayVec3 = [1.0, 1.0, 1.0];
+const UI_Z = 0.001;
+const BUTTON_POSITION: ArrayVec3 = [0, 0, UI_Z];
+
+interface LinkButtonAttrs extends Attrs {
+  linkHoverMenuItem: boolean;
+}
+
+function LinkButton(props: LinkButtonAttrs) {
```

Comment:
I don't feel super strongly about this, and we have done it in some other spots, but I personally wouldn't make a component (in the JSX sense) for this, reads just as clearly with the Button3D inlined. I know some people feel the opposite though.

Reply:
I want to go as is for now, and want to revisit in next or later cycle.